prusti-interface/src/prusti_error.rs

```diff
@@ -40,6 +40,7 @@ pub enum PrustiErrorKind {
     Warning,
     /// A warning which is only shown if at least one error is emitted.
     WarningOnError,
+    Message,
```

Comment:
If there are no Prusti functions that expect `PrustiErrorKind` to always be causes by an error, at this point I think we should rename `PrustiErrorKind` to something like `PrustiDiagnosticKind`.